### PR TITLE
Fix #71: Change property mwfact to master_width_factor

### LINKED ISSUE #71: mwfact renamed to master_width_factor
Elv13: 

> `mwfact` was [renamed](https://www.reddit.com/r/awesomewm/comments/5rk7ek/tyrannical_mwfact_setting_doesnt_effects/) `master_width_factor` upstream. I will have to patch Tyrannical to rename it. Use master_width_factor now, but not the tyrannical.settings. variant. 

If I'm using `maste_width_factor=0,7` where firefox is placed as master and I restore the session the scroll bar has not the correct margin as you can see [here](https://ibin.co/3B3RsReEBFz3.png).

### PR DESCRIPTION
**Pull request by tomoyat1**
I made the changes with sed(1). It works for me, but I would like this to be tested further.

### CODE REVIEW

File: example.rc.lua [hunk 1]
```diff
@@ -381,9 +381,9 @@ globalkeys = awful.util.table.join(
   awful.key({ modkey, "Shift"   }, "q", awesome.quit,
     {description = "quit awesome", group = "awesome"}),
 
-  awful.key({ modkey,           }, "l",     function () awful.tag.incmwfact( 0.05)          end,
+  awful.key({ modkey,           }, "l",     function () awful.tag.incmaster_width_factor( 0.05)          end,
```
Comment (Elv13) [inline example.rc.lua]: That function doesn't exist.

File: example.rc.lua [hunk 2]
```diff
@@ -381,9 +381,9 @@ globalkeys = awful.util.table.join(
...
     {description = "increase master width factor", group = "layout"}),
-  awful.key({ modkey,           }, "h",     function () awful.tag.incmwfact(-0.05)          end,
+  awful.key({ modkey,           }, "h",     function () awful.tag.incmaster_width_factor(-0.05)          end,
```
Comment (Elv13) [inline example.rc.lua]: That function doesn't exist.

File: init.lua [hunk 3]
```diff
@@ -184,12 +184,14 @@ end
 --Match client
 local function match_client(c, forced_tags, hints)
     -- Don't prevent tags from being drag and dropped between screens
+    --[[
```
Comment (Elv13) [inline init.lua]: Unrelated to the commit (will also make a lot of people angry due to the workflow change)

**Comment by tomoyat1** [comment]
Well, that was too sloppy...
Thank you for finding those, I'll fix them shortly.

**Comment by tomoyat1** [comment]
I've fixed the patch.

**Comment by Elv13** [comment]
Thanks

**Pull request merged**